docs/reference/contrib/postgres_search.rst

```diff
@@ -13,23 +13,20 @@ This contrib module provides a search engine backend for Wagtail using
 
 **Features**:
 
-- Supports all the search features available in Wagtail.
+- It supports all the search features available in Wagtail.
 - Easy to install and adds no external dependency or service.
-- Excellent performance for sites with up to 200 000 pages.
-  Stays decent for sites up to a million pages.
-- Faster to reindex than Elasticsearch if you use PostgreSQL 9.5 or more.
+- Excellent performance for sites with up to 200 000 pages and stays decent for sites up to a million pages.
```

Comment:
Do we have any data to support this?

Reply:
ping @BertrandBordage

Reply:
Yes, I made a test using data from a ~ 200 000 pages Wagtail site from Torchbox that @gasman handed me during the Wagtail space where we created this. Response time was still under a second for search queries on those 200 000 pages, including ranking (which is the most expensive operation).
I then extrapolated the data by cloning existing pages until I reached 2 millions if I remember correctly. It was struggling a bit after around a million pages, where response time went over 5 seconds.

But I don’t have a more detailed report nor did a clean script to create “realistic” fake data of that size.

Also, I deleted the data, I don’t remember which site it was from, as it didn’t matter that much.

So if you prefer being cautious about this, you can remove this claim. Even though it was true at that time, it may not stay true in the future.